src/stats/overall.ts

```diff
@@ -67,6 +69,23 @@ export function generateOverallStats(
   return overall;
 }
 
+function determineWinner(playerStocks: StockType[], opponentStocks: StockType[]) : number {
+
+  if (playerStocks.length == 0 && opponentStocks.length == 0) return -1; //unknown winner;
```

Comment:
can we avoid returning undefined by taking out the if statement and moving the `return -1` to the end of the function, will avoid weird edge cases or missing data errors later.

Reply:
Sounds good!